config/README.md

```diff
@@ -0,0 +1,119 @@
+# Dependencies installation
+
+For a faster creation of the environment (and Conda environments in general),
+you can also install [Mamba][mamba] on top of Conda. In that case, replace
+`conda` with `mamba` in the commands below (particularly in 
+`conda env create`).
+
+Create and activate the virtual environment with the required dependencies
+with Conda:
+
+```bash
+conda env create -f environment.yml
+conda activate mirflowz
+```
+
+If you plan to run _MIRFLOWZ_ via Conda, we recommend using the following
+command for a faster environment creation specially if you will run it on an
+HPC cluster.
+
+```bash
+conda config --set channel_priority strict
+```
+In addition, if you want to run _MIRFLOWZ_ via Singularity and do not already
+have it installed globally on your system, you must further update the Conda
+environment with:
+
+```bash
+conda env update -f environment.root.yml
+```
+
+> Mind that you must have the environment activated and root permissions on
+> your system to install Singularity.
```

Comment:
You could consider adding this as a subsection `Running _MIRFLOWZ_ with Singularity`.